File: Backend/Core/Entities/Organization.cs
using System;

namespace Core.Entities
{
    /// <summary>
    /// Thực thể Đơn vị công tác
    /// </summary>
    /// Created by Phuong 25/05/2026
    public class Organization
    {
        public Guid OrganizationId { get; set; }
        public string OrganizationName { get; set; } = string.Empty;
        public Guid? OrganizationParentId { get; set; }
    }
}

File: Frontend/src/models/Organization.js
/**
 * Đại diện cho một đơn vị công tác.
 */
class Organization {
  constructor(organizationId, organizationName, organizationParentId, children = []) {
    this.organizationId = organizationId
    this.organizationName = organizationName
    this.organizationParentId = organizationParentId
    this.children = children
  }
}

export default Organization
